avm-transpiler/src/transpile.rs

```diff
@@ -41,7 +41,11 @@ pub fn brillig_to_avm(brillig: &Brillig) -> Vec<u8> {
                 avm_instrs.push(AvmInstruction {
                     opcode: avm_opcode,
                     indirect: Some(ALL_DIRECT),
-                    tag: Some(AvmTypeTag::FIELD),
+                    tag: if avm_opcode == AvmOpcode::FDIV {
+                        None
```

Comment:
@fcarreiro We define FDIV without any tag? In this way, we can save the tag in the serialization. That's good.

Reply:
@fcarreiro All good. I did not see the first comment above.